code/modules/projectiles/guns/ballistic/rifle.dm

```diff
@@ -143,3 +143,66 @@ obj/item/gun/ballistic/rifle/attackby(obj/item/A, mob/user, params)
 		user.put_in_hands(gun)
 	else
 		user.dropItemToGround(src, TRUE)
+
+///////
+//AWP//
+///////
+
+/obj/item/gun/ballistic/automatic/sniper_rifle/alt
+	name = "AWP"
+	desc = "A powerful sniper rifle chambered in .338 lapua magnum. It kinda smells like tears of silvers, whatever that means."
```

Comment:
Also I don't know if this is a reference to something but it doesn't make sense grammatically anyway, needs to be "silver" not "silvers".

Reply:
Silvers as in the silver rank of CS GO, not everything i do is wrong.

Reply:
silvers are a group of people @Ktlwjec1 know ur stuff